nuxwdog.spec

```diff
@@ -1,14 +1,20 @@
+################################################################################
 Name:           nuxwdog
-Version:        1.0.5
-Release:        2%{?dist}
+################################################################################
 Summary:        Watchdog server to start and stop processes, and prompt for passwords
 # The entire source code is LGPLv2 except for the perl module, which is GPL+ or Artistic
+URL:            http://www.dogtagpki.org/wiki/Nuxwdog
 License:        LGPLv2 and (GPL+ or Artistic)
-URL:            http://www.redhat.com/certificate_system
+
+Version:        1.0.5
+Release:        2%{?_timestamp}%{?_commit_id}%{?dist}
 
```

Comment:
Add this line to show how to define the _phase macro if needed:
```
# global         _phase -a1
```
See https://github.com/dogtagpki/jss/blob/master/jss.spec#L11